nemo/collections/common/data/lhotse/dataloader.py

```diff
@@ -127,6 +129,9 @@ def get_lhotse_dataloader_from_config(
     # Duration filtering, same as native NeMo dataloaders.
     cuts = cuts.filter(DurationFilter(config.min_duration, config.max_duration))
 
+    # Expands cuts if multiple translations are provided.
+    cuts = CutSet(LazyFlattener(cuts.map(_flatten_paired_text)))
```

Comment:
Note to myself: we should add a config field that tells lhotse which functions to apply as part of manifest pre-processing on the fly (such as this one)